docs/documentation/components/navbar.html

```diff
@@ -819,7 +819,7 @@ <h4 class="message-header">Javascript toggle</h4>
           {% highlight html %}<nav class="navbar is-fixed-top">{% endhighlight %}
         </li>
         <li>
-          Add the corresponding <code>has-navbar-fixed-top</code> or <code>has-navbar-fixed-bottom</code> modifier to the <code>&lt;html&gt;</code> element to provide the appropriate padding to the page
+          Add the corresponding <code>has-navbar-fixed-top</code> or <code>has-navbar-fixed-bottom</code> modifier to the <code>&lt;body&gt;</code> element to provide the appropriate padding to the page
```

Comment:
This should be "to either the <code>&lt;html&gt;</code> or the <code>&lt;body&gt;</code> element".

Reply:
Thanks for the review @jgthms !
@Gerhut can you make the change so @jgthms can merge? 😄